js/components/widgets/image-picker.vue

```diff
@@ -129,9 +128,9 @@ export default {
         }
     },
     methods: {
-        save: function() {
+        save() {
             if (this.HAS_FILE_API) {
-                let data = {};
+                const data = {};
```

Comment:
`const` doesn't mean this a constant but that the variable reference is constant ie. the identifier cannot be reassign.
So in case of `Array` or `Object`, if you affect inner members, you still should use `const` if you don't reaffect the variable itself. It would have been wrong if there were `data = {}` somewhere.

I'm just applying the linter recommandations ;)